src/config/settings/base.py

```diff
@@ -82,6 +83,22 @@
     "allauth",
     "allauth.account",
     "allauth.socialaccount",
+    # Machina dependencies:
```

Comment:
Would appreciate a comment linking to docs! and/or having this fully written out as Django Machina

thus the thing being referenced here (django-machina) is written as unambiguous as it could reasonably be.

There should also be a link to a document that details django machina's use in chicage

Reply:
Would you mind going into a bit more detail on what sort of information a document "detailing django-machina's use in chigame" would contain?

Reply:
yes! 
specifically this kind of page should include:
- what django machina is (links to relevant documentation and/or repos)
- why we chose to use it
- how you can see and use the demo
- what some important consequences of using the library are 
  - ie: 
    - does machina not support some feature we want
    - what features of our forum-language/vocab were decided by machina?
- what details are relevant for it's installation? 
  - is there an install guide? 
  - do settings need to be updated? (could be included / referenced when linking an installation guide)
- are there any known issues or bugs that have impacted us?
  - "can't work on X version of python / django"

Reply:
you could reasonably include more or less, the why what and how are the 3 most important.

Reply:
I've open a new issue to address this.
- #270